c/parallel/src/nvrtc/command_list.h

```diff
@@ -229,7 +229,8 @@ struct nvrtc2_pre_build
       std::remove_if(compile_args.args, compile_args.args + compile_args.num_args, [](const char* ptr) -> bool {
         return (ptr == nullptr);
       }));
-    nvrtcResult result = nvrtcCompileProgram(context.program, n_actual_args, compile_args.args);
+    int num_options    = static_cast<int>(n_actual_args);
+    nvrtcResult result = nvrtcCompileProgram(context.program, num_options, compile_args.args);
```

Comment:
Suggestion:
```suggestion
    const int num_options    = static_cast<int>(n_actual_args);
    nvrtcResult result = nvrtcCompileProgram(context.program, num_options, compile_args.args);
```

Reply:
Or simply?
```suggestion
    nvrtcResult result = nvrtcCompileProgram(context.program, static_cast<int>(n_actual_args), compile_args.args);
```

Reply:
Sure, I just assumed @tpn wanted to have a separate variable.